### ISSUE TITLE
Confusion with input tag in basicMarkupHtmlTags

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#213088]
This list says `List of safe, non-input html tags.`, but then 'input' was added to it, and I'm not sure whether that is a problem.

https://github.com/microsoft/vscode/blob/e5dfe045dbed2ebd6e7427e53e40a9f935b50318/src/vs/base/browser/dom.ts#L1863

**Comment by alexr00** [comment]
I will defer to @mjbvz on this!

**Comment by mjbvz** [comment]
@alexr00 I don't think we should support it in hovers and other basic markdown places. Do you remember what feature it was originally added for?

**Comment by alexr00** [comment]
Rendering read-only checkboxes in the comments widget.

**Comment by mjbvz** [comment]
I fixed this a while back as part of my big md refactoring / sanitization pass
